entries/SWC-135.md

```diff
@@ -0,0 +1,17 @@
+# Title
+Statements With No Effects
+
+## Relationships
+[CWE-561: Dead Code](https://cwe.mitre.org/data/definitions/561.html)
```

Comment:
Is https://cwe.mitre.org/data/definitions/1164.html a more accurate CWE relationship?

Reply:
I think so.